calculator-app/src/main/java/camp/nextstep/edu/calculator/CalculatorViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package camp.nextstep.edu.calculator
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import camp.nextstep.edu.calculator.domain.Calculator
+import camp.nextstep.edu.calculator.domain.Expression
+import camp.nextstep.edu.calculator.domain.Operator
+
+class CalculatorViewModel : ViewModel() {
+    private val _calcText = MutableLiveData<String>("")
+    val calcText: LiveData<String> get() = _calcText
+
+    private val _calculatorErrorMessage = SingleLiveEvent<String>()
+    val calculatorErrorMessage: LiveData<String> get() = _calculatorErrorMessage
+
+    private var expression: Expression = Expression.EMPTY
+
+    private val calculator = Calculator()
+
+    fun addToExpression(operand: Int) {
+        val newExpression = expression + operand
+        expression = newExpression
```

Comment:
Expression 에는 operator 구현되어 있기 때문에 다음과 같이 할 수 있습니다.
```suggestion
       expression += operand
```